.github/workflows/publish_docs.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Publish docs'
```

Comment:
Looks like we won't be able to use Github Actions for this because of org legacy reasons.  However if we really want to explore this option we can do something like integrate with Netlify or something else, but this would have to happen in a broader discussion.  We can still update the docs with the other changes, but let's remove this workflow file.